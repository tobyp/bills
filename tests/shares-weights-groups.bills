#!./bills
A	(BC)(DEF).	12.00
