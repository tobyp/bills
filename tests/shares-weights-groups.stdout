todo: B pays 3.0 to A
todo: C pays 3.0 to A
todo: D pays 2.0 to A
todo: E pays 2.0 to A
todo: F pays 2.0 to A
